analytics/serializers.py

```diff
@@ -1,48 +1,51 @@
 from rest_framework import serializers
-from analytics.models import Experiment, LungmapImage
+from analytics.models import Experiment, LungmapImage, ProbeExperiments
+from django.contrib.auth.models import User
 
+# class UserSerializer(serializers.ModelSerializer):
+#     snippets = serializers.PrimaryKeyRelatedField(many=True, queryset=Experiment.objects.all())
+#
```

Comment:
Why comment out code just to add nearly the same thing? Just edit it, it's under VC :)